chain/actors/policy/policy.go

```diff
@@ -685,6 +691,66 @@ func GetAddressedSectorsMax(nwVer network.Version) (int, error) {
 	}
 }
 
+func getPoStedPartitionsMax(nwVer network.Version) (int, error) {
```

Comment:
IMO, this isn't really necessary. It's "correct", but we could also just limit to `miner12.PoStedPartitionsMax` for all actor versions 0-12 without actually breaking anything.

Reply:
Hmm, I'm not sure how much I like that -- it is _wrong_ to do so. But I'm also very uninterested in "supporting" earlier network versions, so I'm not opposed to making that change. 

Your proposed change also has the nice advantage that SPs will start respecting the v12 limit when they upgrade to this release, which reduces the risk of inflight PoSts getting funky over the upgrade.

So...yeah, let's do it.